src/i2c.rs

```diff
@@ -15,6 +13,7 @@ use crate::hal::blocking::i2c::{Read, Write, WriteRead};
 use crate::pac::{DWT, I2C1, I2C2, I2C3};
 use crate::rcc::{sealed::RccBus, Clocks, Enable, GetBusFreq, Reset};
 use crate::time::Hertz;
+use micromath::F32Ext;
```

Comment:
Not a big deal, but there doesn't seem to be any reason for moving this. If so, it would be better to leave this as is, so keep the change small.

Reply:
its cargo format, changing order,

Reply:
Alright then, whatever.